packages/shorebird_cli/lib/src/auth/auth.dart

```diff
@@ -201,11 +274,16 @@ class Auth {
 
   String? get email => _email;
 
-  bool get isAuthenticated => _email != null;
+  bool get isAuthenticated => _email != null || _token != null;
```

Comment:
I think we're going to end up with a list of authenticators which we'll just check in order.